code/modules/loot/loot.dm

```diff
@@ -0,0 +1,120 @@
+/**
+ * Datum for spawning multiple different atoms or applying some special spawning behavior
+ */
+/datum/loot
```

Comment:
I think calling this `/datum/loot` is too generic for what this is actually used for, it should specify that it is for the abandoned crate

Reply:
Or if you hope for this to be used more widely it should be more specific that it is for spawning items inside a container, which `loot` doesn't automatically imply

Reply:
It doesn't have to be a container, can work with a turf or anything else. So it is a generic datum.